cmd/clone.go

```diff
@@ -66,12 +66,12 @@ func Clone(c *git.Client, args []string) error {
 		return err
 	}
 
-	Config(c, []string{"--set", "remote.origin.url", repoid})
```

Comment:
I think it would be good to check the errors coming back from these Config calls even though that wouldn't have fixed this particular problem, there could be other error cases that are missed.

Reply:
No problem.  What about `Fetch` and `Reset` calls below?

Reply:
Fetch doesn't return an error,  Reset should probably be checked

Reply:
I committed it and then amended to add the reset check.